DFC.Digital/DFC.Digital.Service.CourseSearch/Converters/MessageConverter.cs

```diff
@@ -110,14 +128,33 @@ internal static CourseDetails ConvertToCourseDetails(this CourseDetailOutput api
                     LearnerSatisfaction = apiCourseDetail.Provider.FEChoices_LearnerSatisfaction,
                     EmployerSatisfaction = apiCourseDetail.Provider.FEChoices_EmployerSatisfaction
                 },
-                StartDateLabel = apiCourseDetail.Opportunity.FirstOrDefault()?.StartDate.Item,
+                Oppurtunities = GetOppurtunities(apiCourseDetail, activeOppurtunity.OpportunityId),
+                CourseUrl = apiCourseDetail.Course.URL,
                 CourseId = apiCourseDetail.Course.CourseID,
-                AttendanceMode = apiCourseDetail.Opportunity.FirstOrDefault()?.AttendanceMode,
-                AttendancePattern = apiCourseDetail.Opportunity.FirstOrDefault()?.AttendancePattern,
-                QualificationLevel = apiCourseDetail.Course.QualificationLevel,
-                StudyMode = apiCourseDetail.Opportunity.FirstOrDefault()?.StudyMode,
-                Duration = $"{apiCourseDetail.Opportunity.FirstOrDefault()?.Duration?.DurationValue} {apiCourseDetail.Opportunity.FirstOrDefault()?.Duration?.DurationUnit}"
+                Cost = activeOppurtunity.Price,
+                StartDateLabel = activeOppurtunity.StartDate.Item,
+                AttendanceMode = activeOppurtunity.AttendanceMode,
+                AttendancePattern = activeOppurtunity.AttendancePattern,
+                StudyMode = activeOppurtunity.StudyMode,
+                Duration = $"{activeOppurtunity.Duration.DurationValue} {activeOppurtunity.Duration?.DurationUnit}"
             };
         }
+
+        private static List<Oppurtunity> GetOppurtunities(CourseDetailStructure apiCourseDetail, string oppurtunityId)
```

Comment:
Kindly review this, don't believe we need this additional method. I believe we discussed this before?!

Reply:
Removed the additional method